src/Jackett.Common/Indexers/Feeds/AnimeTosho.cs

```diff
@@ -60,11 +60,11 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
         protected override ReleaseInfo ResultFromFeedItem(XElement item)
         {
             var release = base.ResultFromFeedItem(item);
-            var enclosures = item.Descendants("enclosure").Where(e => e.Attribute("type").Value == "application/x-bittorrent");
-            if (enclosures.Any())
+            var enclosure = item.Descendants("enclosure").FirstOrDefault(e => e.Attribute("type").Value == "application/x-bittorrent");
+            if (enclosure != null)
             {
-                var enclosure = enclosures.First().Attribute("url").Value;
```

Comment:
Previously `Where` statement was executed twice (in the worst case scenario) - the first time it was forced by `Any` and the second time by `First`.
Now we search once using `FirstOrDefault` and check the result